install.sh

```diff
@@ -51,17 +51,22 @@ else
 	mount ${DISK}1 ${MOUNT_PATH}
 fi
 
+# chroot into target
 pacstrap ${MOUNT_PATH} base
-arch-chroot ${MOUNT_PATH} /bin/bash <<EOF
```

Comment:
will the commands below this line be executed within the chroot without the redirect? if it does work, we probably need to remove the `EOF` at the very end of this file.

Reply:
This issue has been temporarily dealt with.

We need to test this out.